### PR TITLE
Update contact_utils.py

### PR DESCRIPTION
**Pull request by yuxuanzhuang**
fix bug related to issue #59

### CODE REVIEW

File: contact_calc/contact_utils.py [hunk 1]
```diff
@@ -640,15 +640,15 @@ def extract_ligand_features(top, traj, index_to_atom):
         '''
         if index_to_atom[atom_idx].element == 'C' and len(neighbor_indices) == 3:
```
Comment (RasmusFonseca) [inline contact_calc/contact_utils.py]: Shouldn't this also be `neighbors`?

**Comment by RasmusFonseca** [comment]
Thanks for PR! Added a question as a comment in the code

File: contact_calc/contact_utils.py (hunk 1 above)
Comment (yuxuanzhuang) [reply contact_calc/contact_utils.py]: yes! sorry I missed it.

**Comment by RasmusFonseca** [comment]
Hey @yuxuanzhuang . Would love to merge this, but you never committed your resolution. If you still have this somewhere that'd be great.

**Comment by RasmusFonseca** [comment]
Thanks!!

**Pull request merged**